src/main/scala/is/hail/annotations/Annotation.scala

```diff
@@ -182,4 +182,28 @@ object Annotation {
       case _ => a
     }
   }
+
+  def isSafe(typ: Type, a: Annotation): Boolean = {
+    a == null | (typ match {
```

Comment:
This should be a double bar.

https://stackoverflow.com/questions/5564410/difference-between-and